src/e2e/java/uk/gov/hmcts/reform/sscs/functional/sya/SubmitDraftTest.java

```diff
@@ -25,6 +27,9 @@
     @Value("${test-url}")
     private String testUrl;
 
+    @Value("${test-oauth2Token}")
+    String oauth2Token;
```

Comment:
we cannot inject a hardcoded token. This will fail in the pipeline since that user does not exist in AAT. You have to use a user that exists in AAT. We have created this one: sscs-citizen@hmcts.net. 
Check with me for the password. This user needs to be configured in the properties files.

Reply:
addressed.